### ISSUE TITLE
Is it possible to use mmsegmentation for pixel-wise prediction other than semantic segmentation?

### ISSUE THREAD

**Issue by man469** [open-mmlab/mmsegmentation#3668]
I am currently working on a project, which is very similar to sematic segmentation problem. The difference lies in the output. For semantic segmentation, the output is a pixel-wise image with the same size as the input where each pixel is an int number representing the class index. My project also wants to output a pixel-wise image with the same size as the input, but each pixel is a float number and the loss function is similar to MSE rather than cross entropy in semantic segmentation. Is it possible to realize above function based on mmsegmentation? Thank you for help!

**Comment by AI-Tianlong** [comment]
你说的这个，是不是没有经过后处理的 seg_logits.
经过argmax的后处理，才会变成int的索引，否则的话，只是经过softmax的 浮点数特征图
